.github/workflows/lint.yml

```
@@ -0,0 +1,47 @@
+name: "PR Lint Check"
+
+# PR이 생성되거나 업데이트될 때마다 실행
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  lint-backend:
+    name: Backend Lint
+    runs-on: ubuntu-latest
+    working-directory: ./backend          # 만약 monorepo라면 backend 서브폴더 지정
```

actionlint (1.7.7)

unexpected key "working-directory" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)